### PR TITLE
Fix create login endpoint

### PR DESCRIPTION
**Pull request by davydocsurg**
## Describe your changes
Created the login endpoint and controller

## Issue ticket number and link
#704

### CODE REVIEW

File: __tests__/auth.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { loginDeveloper } from './shared/commands';
+
+describe('login', () => {
+  it('should hit the login endpoint and return a message', async () => {
+    const data = {
+      email: 'test@email.com',
+      password: 'password',
+    };
+    const login = await loginDeveloper(data);
```
Comment (ijemmao) [inline __tests__/auth.test.js:9]: how does this test work? wouldn't you need to create a new developer account in the test with this email and password in order for it to exist in the database?

**Review by ijemmao — COMMENTED** [review]
looking really good! just one comment

File: __tests__/auth.test.js (hunk 1 above)
Comment (davydocsurg) [reply __tests__/auth.test.js:11]: Done

File: __tests__/developers.test.js [hunk 2]
```diff
@@ -102,27 +102,20 @@ describe('Developers', () => {
 
     it('should return developer document with correct credentials', async () => {
       const developerRes = await createDeveloper(developerData);
+      expect(developerRes.status).toEqual(200);
+      console.info(developerRes.body);
       const developerDetails = await getDeveloper({ apiKey: developerRes.body.apiKey });
+      console.log(developerDetails.body, 'test');
```
Comment (ijemmao) [inline __tests__/developers.test.js:108]: remove

File: __tests__/developers.test.js [hunk 3]
```diff
@@ -102,27 +102,20 @@ describe('Developers', () => {
 
     it('should return developer document with correct credentials', async () => {
       const developerRes = await createDeveloper(developerData);
+      expect(developerRes.status).toEqual(200);
+      console.info(developerRes.body);
```
Comment (ijemmao) [inline __tests__/developers.test.js:106]: remove

File: __tests__/__mocks__/documentData.js [hunk 4]
```diff
@@ -10,14 +10,15 @@ const developerData = {
   password: 'password',
 };
 
+const developerData2 = {
```
Comment (ijemmao) [inline __tests__/__mocks__/documentData.js:13]: it's not great practice to include numbers in variable names - please change this variable name

**Review by ijemmao — CHANGES_REQUESTED** [review]
can you also attach a screenshot of the test suite passing?

**Comment by davydocsurg** [comment]
> can you also attach a screenshot of the test suite passing?

Screenshot 2023-09-08 at 6 48 59 AM

**Review by ijemmao — APPROVED** [review]

**Pull request merged**